index.html

```diff
@@ -504,12 +501,12 @@ <h4 data-i18n="index.page.requirements.updates.header">Frequent Updates</h4>
             }
 
             // update features (limit 5)
-            var feature_content = "<h2>" + i18n.t("index.page.content.new-features.new-features-in", { defaultValue: "New in " + build.versionString, sprintNumber: build.versionString }) + "</h2>";
-            build.newFeatures.slice(0, Math.min(build.newFeatures.length, 5)).forEach(function (feature) {
-                feature_content += "<h3>" + $("<div/>").text(feature.name).html() + "</h3>";
-                feature_content += "<p>" + $("<div/>").text(feature.description).html() + "</p>";
+            var feature_content = '<h2>' + i18n.t("index.page.content.new-features.new-features-in", { defaultValue: "New in " + build.versionString, versionString: build.versionString }) + '</h2>';
+            build.newFeatures.slice(0, 5).forEach(function (feature) {
+                feature_content += '<h3>' + $("<div/>").text(feature.name).html() + '</h3>';
+                feature_content += '<p>' + $("<div/>").text(feature.description).html() + '</p>';
```

Comment:
Why the change from double quote (`"`) to single quote (`'`). I see why on line [509](https://github.com/adobe/brackets.io/pull/139/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR509), but I don't see why here. (My assumption on 500 is that you were trying to use double quotes for the attributes).